crates/db_views/src/comment_view.rs

```diff
@@ -449,9 +441,21 @@ impl<'a> CommentQueryBuilder<'a> {
               .or(community_follower::person_id.eq(person_id_join)),
           )
         }
-        ListingType::Community => {}
-      };
-    }
+        ListingType::Community => {
+          if self.community_actor_id.is_none() && self.community_id.is_none() {
```

Comment:
The Post and Comment queries now check do this community check in a better place.